tests/integration/components/contributors/component-test.ts

```diff
@@ -122,6 +122,37 @@ module('Integration | Component | contributors', hooks => {
         assert.dom(`[data-test-contributor-permission="${contributor.id}"]`)
             .hasText('Read');
         assert.dom('[data-test-contributor-citation-checkbox]').isChecked();
+        assert.dom(`[data-test-contributor-remove="${contributor.id}"]`).exists();
+    });
+
+    test('editable contributor card can remove contributor', async function(assert) {
+        const firstContributor = server.create('contributor', {
+            fullName: 'First Contributor',
+            index: 0,
+            id: 'Keep',
+        });
+        const secondContributor = server.create('contributor', {
+            fullName: 'Second Contributor',
+            index: 1,
+            id: 'Remove',
+        });
+        const draftRegistration = server.create('draft-registration', {
+            contributors: [firstContributor, secondContributor],
+        });
+        const registrationModel = await this.store.findRecord('draft-registration', draftRegistration.id);
+        this.set('node', registrationModel);
+        await render(hbs`<Contributors::Widget @node={{this.node}} @widgetMode={{'editable'}} />`);
+        const elementsBefore = findAll('[data-test-contributor-card]');
+        assert.equal(elementsBefore[0].getAttribute('data-test-contributor-card'), 'Keep');
+        assert.equal(elementsBefore[1].getAttribute('data-test-contributor-card'), 'Remove');
```

Comment:
I can understand using `findAll` and `element.getAttribute()` for checking the rendering order. Since the ordering of contributors is tested separately, I think just using `assert.dom()` is preferable as it fails better (by failing the assert). If `findAll` didn't return anything for whatever reason, we'd get `typeError` .
```suggestion
        assert.dom('[data-test-contributor-card="Keep"]').isVisible();
        assert.dom('[data-test-contributor-card="Remove"]').isVisible();
```

Reply:
Yeah, I apparently wasn't thinking there.